pdr_backend/pdr_dashboard/callbacks/callbacks_home.py

```diff
@@ -62,7 +63,7 @@ def get_display_data_from_db(
             payouts = app.data.payouts(
                 [row["contract"] for row in selected_feeds],
                 predictoors_addrs,
-                start_date,
+                UnixTimeMs(start_date * 1000) if start_date else None,
```

Comment:
1000 :)

Reply:
this is still here. Please remember no multiplication or division with 1000 outside of time utils